platform/javascript/javascript_main.cpp

```diff
@@ -47,9 +48,18 @@ void exit_callback() {
 }
 
 void main_loop_callback() {
+	uint64_t current_ticks = os->get_ticks_usec();
+
 	bool force_draw = DisplayServerJavaScript::get_singleton()->check_size_force_redraw();
 	if (force_draw) {
 		Main::force_redraw();
+	} else if (current_ticks < target_ticks && !force_draw) {
```

Comment:
The `&& !force_draw` part is unnecessary, you can't enter this `else if` condition if `force_draw` was true.

Reply:
Yep, so much rebasing ;). Ty